lua/rainbow/internal.lua

```diff
@@ -68,11 +69,11 @@ function M.attach(bufnr, lang)
     return
   end
 
-  callbackfn(bufnr) -- do it on intial load
+  callbackfn(bufnr) -- do it on attach
   vim.api.nvim_buf_attach( --do it on every change
     bufnr,
     false,
-    { on_lines = try_async(callbackfn(bufnr))}
+    { on_lines = try_async(callbackfn)}
```

Comment:
why did you remove the bufnr argument?

Reply:
changing it to
```lua
 on_lines = try_async(callbackfn, bufnr)}
```
slows it down again

Reply:
@sunjon just to be clear, was this removal intentional? `try_async` was defined with 2 args and adding the 2nd `bufnr` argument fixes the error